src/containers/shared/components/Transaction/NFTokenAcceptOffer/Simple.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+import { useTranslation } from 'react-i18next'
+import Account from '../../Account'
+import { SimpleRow } from '../SimpleRow'
+import { Amount } from '../../Amount'
+import { NFTokenAcceptOfferInstructions } from './types'
+import { TransactionSimpleComponent, TransactionSimpleProps } from '../types'
+
+export const Simple: TransactionSimpleComponent = ({
+  data,
+}: TransactionSimpleProps<NFTokenAcceptOfferInstructions>) => {
+  const { acceptedOfferIDs, amount, tokenID, seller, buyer } =
+    data?.instructions
+  const { t } = useTranslation()
+
+  return (
+    <>
+      {acceptedOfferIDs.map((offer) => (
+        <SimpleRow label={t('offer_index')} className="dt" data-test="offer-id">
+          {offer}
+        </SimpleRow>
+      ))}
+      {amount && seller && buyer && tokenID && (
+        <>
+          <SimpleRow label={t('seller')} className="account" data-test="seller">
+            <Account account={seller} />
+          </SimpleRow>
+          <SimpleRow label={t('buyer')} className="account" data-test="buyer">
```

Comment:
i dont think the className needs to be set on these two.  The className is already set inside `<Account>` and putting it on the row doesn't appear to do anything looking at the styles.